staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -25,14 +25,17 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
-bash "${CODEGEN_PKG}/generate-groups.sh" all \
-  k8s.io/sample-apiserver/pkg/generated k8s.io/sample-apiserver/pkg/apis \
+"${CODEGEN_PKG}/generate-groups.sh" all \
+  k8s.io/sample-apiserver/pkg/generated \
+  k8s.io/sample-apiserver/pkg/apis \
   "wardle:v1alpha1,v1beta1" \
   --output-base "$(dirname "${BASH_SOURCE[0]}")/../../.." \
   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate.go.txt
 
-bash "${CODEGEN_PKG}/generate-internal-groups.sh" "deepcopy,defaulter,conversion,openapi" \
-  k8s.io/sample-apiserver/pkg/generated k8s.io/sample-apiserver/pkg/apis k8s.io/sample-apiserver/pkg/apis \
+"${CODEGEN_PKG}/generate-internal-groups.sh" "deepcopy,defaulter,conversion,openapi" \
+  k8s.io/sample-apiserver/pkg/generated \
+  k8s.io/sample-apiserver/pkg/apis \
+  k8s.io/sample-apiserver/pkg/apis \
```

Comment:
why the same path twice?

Reply:
I see 
```
Usage: $(basename $0) <generators> <output-package> <internal-apis-package> <extensiona-apis-package> <groups-versions> ...
```

Reply:
Yeah, I hate these scripts, but ... shell...